yt/yt/tests/integration/proxies/test_rpc_proxy.py

```diff
@@ -853,6 +853,29 @@ def test_sort(self):
 
         assert select_rows("* from [//tmp/t_out] LIMIT " + str(2 * size)) == original_table
 
+    @authors("faucct")
+    def test_writing_large_rows(self):
+        data = [{"index": 0, "str": "F" * (17 << 20)}]
+        create("table", "//tmp/table", attributes={"dynamic": False, "schema": self._schema})
+        write_table("//tmp/table", data, table_writer={"max_row_weight": 20 << 20})
```

Comment:
Let's also test the static table limits, that they are cut off at 128M. My only concern is how long a test with 128M + eps of data will run, but I have no ideas on how to improve it yet.